contrib/udfs/src/test/java/org/apache/drill/exec/udfs/TestTimeBucketFunction.java

```diff
@@ -92,6 +94,28 @@ public void testTimeBucket() throws Exception {
       .go();
   }
 
+  @Test
+  public void testDoubleTimeBucket() throws Exception {
+    String query = "SELECT time_bucket(CAST(1451606760 AS DOUBLE), 300000) AS high FROM (values(1))";
+    testBuilder()
+      .sqlQuery(query)
+      .ordered()
+      .baselineColumns("high")
+      .baselineValues(1451400000L)
+      .go();
+  }
+
+  @Test
+  public void testTimeBucketTimestamp() throws Exception {
+    String query = "SELECT time_bucket(CAST(1585272833845 AS TIMESTAMP), 300000) AS high FROM (values(1))";
```

Comment:
Could you please replace it with string representation, it looks more friendly and helps to understand the source value:
```suggestion
    String query = "SELECT time_bucket(timestamp '2020-03-27 01:33:53.845', 300000) AS high";
```

Also, there is no need to specify `FROM (values(1))` if it is not used.

Reply:
Removed.